sleap_roots/scanline.py

```diff
@@ -0,0 +1,63 @@
+"""Get intersections between roots and horizontal scan lines."""
+
+import numpy as np
+import math
+from shapely import LineString
+
+
+def get_pt_scanline(pts: np.ndarray, depth=1080, width=2048, Nline=50) -> list:
+    """Get intersection points of roots and scan lines.
+    
+    Args:
+        pts: Numpy array of points of shape (instances, nodes, 2).
+        depth: the depth of cylinder, or number of rows of the image.
+        width: the width of cylinder, or number of columns of the image.
+        Nline: number of scan lines.
+        
+    Returns:
+        A list of intersection xy location, with length of Nline, each has shape 
+        (# intersection,2).
+    """
+    # connect the points to lines using shapely
+    points = list(pts)
+    
+    # calculate interval between two scan lines
+    Ninterval = math.ceil(depth/(Nline-1))
```

Comment:
Variable names should not start with uppercase. Try this:

```suggestion
    n_interval = math.ceil(depth / (n_line - 1))
```